src/Model/Subsite.php

```diff
@@ -783,7 +783,7 @@ public function fieldLabels($includerelations = true)
      */
     public function allowedThemes()
     {
-        if ($themes = self::$allowed_themes) {
+        if (($themes = self::$allowed_themes) || ($themes = ThemeResolver::singleton()->getCustomThemeOptions())) {
```

Comment:
Do you want to make a separate PR to master to deprecate `$allowed_themes`?

Reply:
Yes - I was going to deprecate this and another useless extension point after I merge up to `2.3`.